Alexey/Text/reverse_a_string/tests/test_reverse_a_string.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from Alexey.Text.reverse_a_string.main.reverse_a_string import reverse
+
+
+class TestReverseAString(unittest.TestCase):
+    def test_send_sentence_and_expect_palindrome_message(self):
```

Comment:
Please add tests for` '\0\n\t' `and for `'\\0\\n\\t'`

Reply:
done